searchcore/src/vespa/searchcore/proton/server/proton.cpp

```diff
@@ -763,6 +764,10 @@ Proton::updateMetrics(const metrics::MetricLockGuard &)
         metrics.resourceUsage.memoryMappings.set(usageFilter.getMemoryStats().getMappingsCount());
         metrics.resourceUsage.openFileDescriptors.set(FastOS_File::count_open_files());
         metrics.resourceUsage.feedingBlocked.set((usageFilter.acceptWriteOperation() ? 0.0 : 1.0));
+        struct mallinfo mallocInfo = mallinfo();
+        // Vespamalloc reports arena in 1M blocks as an 'int' is too small.
+        // If we use something else than vespamalloc this must be changed.
+        metrics.resourceUsage.mallocArena.set(uint64_t(mallocInfo.arena) << 20);
```

Comment:
Consider using the __Mi_ operator from size_literals.h instead.

Reply:
Fixed